nmrfx-core/src/main/java/org/nmrfx/chemistry/io/PDBFile.java

```diff
@@ -372,38 +351,25 @@ public int checkPDBType(String fileName) throws MoleculeIOException {
                     String resNum = atomParse.resNum;
                     Vector3d vector3 = new Vector3d(atomParse.x, atomParse.y, atomParse.z);
                     atomMap.put(aName, vector3);
-                    boolean gotAtoms = false;
                     if (!resNum.equals(lastRes)) {
                         if (atomMap.containsKey("CA") && atomMap.containsKey("CG")) {
-                            Vector3d vectorCA = (Vector3d) atomMap.get("CA");
-                            Vector3d vectorCG = (Vector3d) atomMap.get("CG");
                             if (atomMap.containsKey("HB1") && atomMap.containsKey("HB2")) {
-                                Vector3d vectorHB1 = (Vector3d) atomMap.get("HB1");
-                                Vector3d vectorHB2 = (Vector3d) atomMap.get("HB1");
-                                double volume = chiralVolume(vectorCA, vectorCG, vectorHB1, vectorHB2);
```

Comment:
volume was unused, which meant many of the other variables were unused, so removed